pkg/chezmoi/chezmoi_unix.go

```diff
@@ -49,6 +52,23 @@ func etcHostnameFQDNHostname(fileSystem vfs.FS) (string, error) {
 	return "", s.Err()
 }
 
+// etcMynameFQDNHostname returns the FQDN hostname from parsing /etc/myname.
+// See OpenBSD's myname(5) for details on this file.
+func etcMynameFQDNHostname(fileSystem vfs.FS) (string, error) {
+	contents, err := fileSystem.ReadFile("/etc/myname")
+	if err != nil {
+		return "", err
+	}
+	s := bufio.NewScanner(bytes.NewReader(contents))
+	for s.Scan() {
```

Comment:
This loop needs to ignore comment lines as well.

Reply:
Done!